services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    new_order = Order.objects.create(
+        user_id=User.objects.get(username=username).id
```

Comment:
according to checklist you need to use get_user_model() instead of User

Reply:
agree